src/ovp_pipeline/commands/backfill_entity_type.py

```
@@ -0,0 +1,259 @@
+    for i, (fp, text, fm) in enumerate(needs_classification):
+        title = fm.get("title", fp.stem.replace("-", " "))
+        body_start = _FRONTMATTER_RE.match(text)
+        body = text[body_start.end() :] if body_start else text
+        definition = ""
+        def_match = re.search(r">\s*\*\*(?:一句话定义|Definition)\*\*:\s*(.+)", body, re.IGNORECASE)
+        if def_match:
+            definition = def_match.group(1).strip()
+
+        try:
+            kind = _classify(llm, title, definition, body[:500])
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)